File: Java/Armstrong.java

import java.util.Scanner;

public class Armstrong {
    public static void main(String[] args) {
       Scanner sc = new Scanner(System.in);
       System.out.println("Enter a number: ");
       int num = sc.nextInt();
       int i = num;
       int sum = 0;
       int count = String.valueOf(num).length();

       while (num > 0){
        int digit = num % 10;
        sum += Math.pow(digit, count);
        num = num / 10;
       }

        if (sum == i) {
            System.out.println("Armstrong Number");
        } else {
            System.out.println("Not Armstrong Number");
        }

    }
}

File: java_to_js/armstrong.js
function armstrong(num){
    let i = num;
    let sum = 0;
    let count = num.length;
    num = Number(num);
    while(num > 0){
        let digit = num % 10;
        sum += Math.pow(digit,count);
        num = Math.floor(num / 10);
    }
    let output = sum + "<br>";
    if(sum == i){
        output += "Armstrong number";
    }
    else{
        output += "Not an Armstrong number";
    }
    return output;
}